src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -174,21 +174,26 @@
             border: 1px solid var(--main-color);
         }
         .table-container {
-            width:  100%;
+            width: 100%;
+            height: 50vh;
```

Comment:
Hardcoding 50vh doen't looks like a perfect fix for all displays. There can be a case when the elements above this table are together occupying space << 50vh. Hence again, the empty space the end can be present.
Or it might overflow if display is small.

Reply:
That is the reason why I chose to also add the toggle for the table scrollbar, so in weird edge cases, people can just deactivate it altogether. Once we have put the user persistence system on a more solid footing and if many users want to have the scrollbar deactivated permanently, we can make the switching off persistent.

Having said this, if you have a better short term solution, let me know!

Reply:
What about using flex on the outer div with height 100vh, and then flex:1 on tx table, 
or getting remaining height using javascript?